frontend/projects/ui/src/environments/environment.ts

```diff
@@ -4,6 +4,7 @@
 
 export const environment = {
   production: false,
+  useServiceWorker: true,
```

Comment:
Is this dev?

Reply:
yeah - well, any environment that is not prod.